viewer/src/views/threejs/cad/CadNode.ts

```diff
@@ -204,6 +205,10 @@ export class CadNode extends THREE.Object3D {
           throw new Error(`Could not find 3D node for sector ${sector.id} - invalid id?`);
         }
         if (sectorNode.group) {
+          sectorNode.group.userData.used = false;
```

Comment:
Could we add some functions to `SectorNode`, e.g. `increaseRefCount()`, `isReferenced()` and `decreaseRefCount()`?